src/components/products/productsCard.jsx

```diff
@@ -34,7 +34,30 @@ const ProductsCard = () => {
       console.error(error);
     }
   };
-
+  const addToCart = (
+    _id,
+    title,
+    productImage,
+    discountPrice,
+    originalPrice,
+    rating
+  ) => {
+    cartItemsDispatch({
+      type: "ADD_TO_CART",
+      payload: {
+        value: {
+          _id,
+          title,
+          productImage,
+          discountPrice,
+          originalPrice,
+          quantity: 1,
+          totalPrice: originalPrice,
+          rating,
```

Comment:
is there need to destructure again can use ```...rest``` instead

Reply:
_yes yes  will try that to improve thanks captain for pointing it out_